docs/victoriametrics/vmalert.md

```diff
@@ -902,6 +906,9 @@ a review to the dashboard.
 Try the following tips to avoid common issues:
 1. Always set [group's interval](https://docs.victoriametrics.com/victoriametrics/vmalert/#groups) to be **equal to or greater than** 
    the [time series resolution](https://docs.victoriametrics.com/victoriametrics/keyconcepts/#time-series-resolution).
+1. Don't set labels with dynamic values to `labels` [param](https://docs.victoriametrics.com/victoriametrics/vmalert/#alerting-rules).
+    * 👉 Example: setting `label: {{$value}}` to the rule will break its [alert state tracking](https://docs.victoriametrics.com/victoriametrics/vmalert/#alert-state)
```

Comment:
Why the 👉 is needed here?

Reply:
To visually distinguish "theory" and "practice" parts. You don't like it? I don't have strong reasoning about it, for me it just looks more clear than without it.